consensus/safety-rules/src/test_utils.rs

```diff
@@ -66,7 +66,7 @@ pub fn make_proposal_with_qc_and_proof(
         Block::new_proposal(
             payload,
             round,
-            duration_since_epoch().as_secs(),
+            qc.certified_block().timestamp_usecs() + 1,
```

Comment:
Why is this change necessary?

Reply:
because I added missing check for `block.verify_well_formed` and it requires strict increasing timestamps